util/expansion.go

```diff
@@ -102,7 +102,15 @@ func (exp *Expansions) ExpandString(toExpand string) (string, error) {
 				return []byte(exp.Get(match))
 			}
 
-			return []byte(defaultVal)
+			// look for an expansion in the secondary value
+			if strings.HasPrefix(secondaryValue, "*") {
+				// trim off *
+				secondaryValue = secondaryValue[1:]
```

Comment:
do we have any precedent for this pattern of appending a * to something to indicate that it should be looked up?

Reply:
There isn't any precedent - the user who submitted the ticket suggested this syntax (among others) and I thought it was cleaner than nesting ${val1|${val2}} while still making it clear that the second value is a variable and not a hard coded string